source/api/discovery/0.3/index.md

```diff
@@ -0,0 +1,784 @@
+
+```
+{ "startTime": "2017-09-20T23:58:00Z" }
+```
+
+##### summary
+
+A short textual description of the Activity. This is intended primarily to be used for debugging purposes or explanatory messages.
+
+Activities _MAY_ have the `summary` property.  The value _MUST_ be a string.
+
+```
+{ "summary": "admin updated the manifest, fixing reported bug #15." }
+```
+
+##### actor
+
+The organization, person, or software agent that carried out the Activity.
+
+Activities _MAY_ have the `actor` property.  The value _MUST_ be a JSON object, with the `id` and `type` properties. The `id` _SHOULD_ be an HTTP(S) URI. The `type` _MUST_ be one of `Application`, `Organization`, or `Person`.
+
+```
+{ 
+  "actor": {
+    "id": "https://example.org/person/admin1",
+    "type": "Person"
+  }
+}
+```
+
+##### Complete Activity Example
+
+A complete example Activity would thus look like the following example. Note that it does not have a `@context` property, as it is always embedded within a `CollectionPage`.
+
+```
+{ 
+  "id": "https://example.org/activity/1",
+  "type": "Update",
+  "summary": "admin updated the manifest, fixing reported bug #15.",
+  "object": {
+    "id": "https://example.org/iiif/1/manifest",
+    "type": "Manifest",  	
+    "seeAlso": [
+      {
+        "id": "https://example.org/dataset/single-item.jsonld",
+        "type": "Dataset",
+        "format": "application/ld+json"
+      }
+    ]
+  },
+  "endTime": "2017-09-21T00:00:00Z",
+  "startTime": "2017-09-20T23:58:00Z",
+  "actor": {
+    "id": "https://example.org/person/admin1",
+    "type": "Person"
+  }
+}
+```
+
+
+### 3.4. Linked Data Context and Extensions
+
+#### 3.4.1. @context
+
+The top level resource in the response _MUST_ have the `@context` property, and it _SHOULD_ appear as the very first key/value pair of the JSON representation.  This property lets Linked Data processors interpret the document as a graph.  The value of the property _MUST_ be either the URI of the IIIF Discovery context document, `http://iiif.io/api/discovery/0/context.json`, or it _MUST_ be an array of strings, where the URI of the IIIF Discovery context document is the last item in the array.  
+
+```
+{
+  "@context": "http://iiif.io/api/discovery/0/context.json"
+}
+```
+
+#### 3.4.2. Extensions
+
+If any additional classes or properties are desired beyond the ones defined in this specification or the ActivityStreams specification, then those classes or properties _SHOULD_ be mapped to RDF terms in one or more additional context documents. These extension contexts _SHOULD_ be added to the top level `@context` property, and _MUST_ be before the discovery context. The JSON-LD 1.1 functionality of defining terms only within a specific property, known as [scoped contexts][org-w3c-json-ld-scoped-contexts], _MUST_ be used to minimize cross-extension collisions.  Extensions intended for broad use _SHOULD_ be registered in the [extensions registry][annex-registry].  
```

Comment:
'Discovery context` (capitalized) or even more verbose (e.g., "URI of..." as in 3.4.1)

Reply:
:+1: